keyboards/reviung/reviung46/keymaps/default/keymap.c

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 gtips (@gtips)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+	_BASE,
+	_LOWER,
+	_RAISE,
+	_ADJUST
+};
+
+#define LOWER  MO(_LOWER)
+#define RAISE  MO(_RAISE)
+#define ADJUST MO(_ADJUST)
+
```

Comment:
These are unused.
```suggestion
```

Reply:
Thanks.
Maybe I don't understand it well enough.
Please let me know.
If I remove lines 14-15, lines 19-20 and 49-51 don't work.

Reply:
Oh, I didn't full see lines 19-20 -- they are incorrect.